Application/EdFi.Ods.Api/OperationalContext/PersonIdentification/NoUniqueIdMatchException.cs

```diff
@@ -28,10 +28,5 @@ public NoUniqueIdMatchException(string message)
 
         public NoUniqueIdMatchException(string message, Exception inner)
             : base(message, inner) { }
-
-        protected NoUniqueIdMatchException(
-            SerializationInfo info,
-            StreamingContext context)
-            : base(info, context) { }
```

Comment:
Fix warning related to [legacy serialization](https://github.com/dotnet/docs/issues/34893)